### DISCUSSION TITLE
Terminal opens and closes

### DISCUSSION THREAD

**Discussion by Kouroshdan** [pyinstaller/pyinstaller#8115] [Help] [answered]
hello , I have a python project that has GUI and Im using tkinter for that . The problem is when i open the .exe file it opens the terminal for one  moment and immediately closses .
Can it be a problem with the code ? If you think sharing the code may be helpful please let me know 
thanks

**Comment by rokm** [comment, marked as answer]
If you open a Command prompt (`cmd`) (or Windows Terminal) and run your exe from there instead of double-clicking on it, you should see the error that caused your program to close.

**Reply by Kouroshdan** [reply]
Im so sorry but I realy dont know how to work with the cmd , I just used google and  I Founded that I should use start [myfile.exe] but when I  it cmd could not find the file so i used the path like , start C:\Users\ , but it just opens the folder of the file . so now im not sure if using start is correct . what do you reccomend using ? 
thank you for your time .

**Reply by rokm** [reply]
You can use Explorer to go the directory with the exe, and then type `cmd .` in the location bar. This should open the command prompt in that directory. Then you just type `myfile.exe` in the command prompt.

**Reply by Kouroshdan** [reply]
found it , it could not find one of the photos used in the GUI 
thanks
